### PR TITLE
Add Bali release schedule

### PR DESCRIPTION
**Pull request by stmcginnis**
This adds a landing page and a proposed release schedule for the OpenSDS
Bali release.

Signed-off-by: Sean McGinnis <sean.mcginnis@gmail.com>

### CODE REVIEW

File: docs/source/bali/index.rst
Comment (xing-yang) [inline docs/source/bali/index.rst]: Have you checked with Steven on publishing this?  I'm not sure if we've made a final decision on this yet.

Comment (stmcginnis) [reply docs/source/bali/index.rst]: It is still being finalized, but I wanted to include it here being optimistic. We should know soon. Steven is added as a reviewer on this PR as well.

If it ends up falling through we can quickly drop it from the schedule.

**Comment by leonwanghui** [comment]
It looks great, but @stmcginnis could you explain a bit about how to display these docs, is it a website or just some formatted files? Thanks!

**Comment by stmcginnis** [comment]
Hey @leonwanghui - yes, this is the source for what gets published to https://releases.opensds.io/

**Comment by leonwanghui** [comment]
Got it, thanks @stmcginnis !

File: docs/source/bali/index.rst
Comment (xing-yang) [inline docs/source/bali/index.rst:27]: Should this be changed to August 7?

Comment (xing-yang) [reply docs/source/bali/index.rst:27]: Oh, never mind.  I see that you said "the week of August 6".

File: docs/source/bali/index.rst
Comment (xing-yang) [inline docs/source/bali/index.rst]: Do you mean no new features may be proposed without the review/approval after milestone 2?  So m-2 is like a feature proposal deadline?

Comment (stmcginnis) [reply docs/source/bali/index.rst]: Correct. I think after this point we would need to have a discussion at the TSC level as to whether any planned feature changes would be of high enough priority to change plans. After milestone-2 we would want to focus just on getting our currently planned features fully implemented and tested.

Comment (xing-yang) [reply docs/source/bali/index.rst]: Can you change to "No new features may be proposed for the Bali release without the review and approval of the Technical Steering Committee (TSC) after the second milestone" to make it clear?

Comment (stmcginnis) [reply docs/source/bali/index.rst]: This is under the second milestone section, so that's pretty much exactly what it currently says.

Maybe looking at the generated output would make this more clear?

Comment (xing-yang) [reply docs/source/bali/index.rst]: Under Milestone 3, you specified that "All new feature development should be complete by the third milestone."  I think it is more clear to specify milestone-2 here.

Comment (stmcginnis) [reply docs/source/bali/index.rst]: Milestone 2 would be way too early for that I think. Would that be possible? Then what would happen between m-2 and m-3?

Comment (xing-yang) [reply docs/source/bali/index.rst]: I didn't mean to change the text for "All new features...".  What I was trying to say is that I like the sentence under Milestone 3 where it clearly stated what needs to be done by m-3.  I'd like to see "after the second milestone" added in this section for "No new features may be proposed" so it is more clear.  I hope I'm not adding more confusion to my comments here:).

**Review by xing-yang — APPROVED** [review]
lgtm

**Pull request merged**
